lib/state_change_metadata.go

```diff
@@ -595,3 +596,41 @@ func (stakeRewardSCM *StakeRewardStateChangeMetadata) GetVersionByte(blockHeight
 func (stakeRewardSCM *StakeRewardStateChangeMetadata) GetEncoderType() EncoderType {
 	return EncoderTypeStakeRewardStateChangeMetadata
 }
+
+type UnjailValidatorStateChangeMetadata struct {
+	ValidatorPKID         *PKID
+	JailedAtEpochNumber   uint64
+	UnjailedAtEpochNumber uint64
+}
+
+func (unjailValidatorSCM *UnjailValidatorStateChangeMetadata) RawEncodeWithoutMetadata(blockHeight uint64, skipMetadata ...bool) []byte {
```

Comment:
Maybe just shorten this to `metadata` for conciseness?
```suggestion
func (metadata *UnjailValidatorStateChangeMetadata) RawEncodeWithoutMetadata(blockHeight uint64, skipMetadata ...bool) []byte {
```